plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js

```diff
@@ -330,19 +330,34 @@ jQuery( function ( $ ) {
 	);
 
 	// Stock options.
+	function show_or_hide_stock_management_fields(
+		isStockManagementEnabled,
+		productType
+	) {
+		const stockManagementFields = $( '.stock_fields' );
+		const stockStatusField = $( '.stock_status_field' );
```

Comment:
I would follow the jQuery naming convention for variables that store jQuery objects prefixing them with a `$`.

Reply:
Good point